src/app/search-bar/search-bar.component.html

```diff
@@ -1,75 +1,67 @@
-        (keyup.escape)="_exitSearch()"
-        type="text"
+      <input [(ngModel)]="searchText" (ngModelChange)="_handleSearchTextChange($event)"
+        (keyup.arrowup)="_handleArrowKey('UP')" (keyup.arrowdown)="_handleArrowKey('DOWN')"
+        (keyup.enter)="_handleCreatorSelect(creators[selectedCreatorIndex])" (keyup.escape)="_exitSearch()" type="text"
         class="form-control shadow-none search-bar__fix-active"
         style="font-size: 15px; padding-left: 0; border-left-color: rgba(0, 0, 0, 0)"
-        [ngClass]="{ 'br-12px': !isSearchForUsersToSendClout }"
-        placeholder="Search"
-      />
+        [ngClass]="{ 'br-12px': !isSearchForUsersToSendClout }" placeholder="Search" />
     </div>
   </div>
 
   <!-- Results Dropdown -->
-  <div class="w-100 search-bar__results-dropdown" *ngIf="searchText != '' && (creators.length > 0 || loading)">
+  <div class="w-100 search-bar__results-dropdown" *ngIf="searchText != ''">
     <div *ngIf="loading" class="d-flex justify-content-center fs-15px fc-muted p-5px">Loading...</div>
-    <div *ngIf="!loading && creators.length > 0" class="fs-15px">
-      <div
-        *ngFor="let creator of creators; let ii = index"
-        class="d-flex align-items-center"
-        [ngClass]="{ 'search-bar__selected-color': creator.Username == creatorSelected }"
-        (click)="_handleCreatorSelect(creators[selectedCreatorIndex])"
-        (mouseover)="_handleMouseOver(creator.Username, ii)"
-      >
-        <div class="search-bar__avatar m-10px" [avatar]="creator.PublicKeyBase58Check"></div>
-        <div>
-          <span>{{ creator.Username || creator.PublicKeyBase58Check }}</span>
-          <span
-            *ngIf="creator.IsReserved && !creator.IsVerified"
-            class="d-inline-block ml-1 cursor-pointer lh-12px fc-muted"
-            matTooltipClass="global__mat-tooltip global__mat-tooltip-font-size"
-            [matTooltip]="'This profile is reserved. Click to learn how to claim it.'"
-          >
-            <i class="far fa-clock fa-md align-middle"></i>
-          </span>
-          <span
-            *ngIf="creator.IsVerified"
-            (click)="tooltip.toggle()"
-            class="ml-1 cursor-pointer text-primary"
-            matTooltipClass="global__mat-tooltip global__mat-tooltip-font-size"
-            [matTooltip]="'This account is verified'"
-            #tooltip="matTooltip"
-          >
-            <i class="fas fa-check-circle fa-md align-middle"></i>
-          </span>
-          <span *ngIf="creator.CoinPriceBitCloutNanos">&nbsp;&middot;&nbsp;</span>
+    <div class="fs-15px">
+      <div *ngIf="!loading && creators.length > 0">
+        <div *ngFor="let creator of creators; let ii = index" class="d-flex align-items-center"
+          [ngClass]="{ 'search-bar__selected-color': creator.Username == creatorSelected }"
+          (click)="_handleCreatorSelect(creators[selectedCreatorIndex])"
+          (mouseover)="_handleMouseOver(creator.Username, ii)">
```

Comment:
With the addition of the new element at the bottom, the last creator that was hovered over is still being displayed as selected. Let's add a mouseout handler.


In search-bar.component.ts
```suggestion
          (mouseover)="_handleMouseOver(creator.Username, ii)"  (mouseout)="_handleMouseOut(creator.Username, ii)">
```
```
  _handleMouseOut(creator: string, index: number) {
    if (this.creatorSelected === creator) {
      this.creatorSelected = "";
    }
    if (this.selectedCreatorIndex === index) {
      this.selectedCreatorIndex = -1;
    }
  }
```

Reply:
Good catch! At the moment, this happens for all results in the popdown. We can add a mouseout for each one so the result never stays highlighted when you leave that specific row.